src/modules/GameConstants.ts

```diff
@@ -1579,6 +1579,39 @@ export enum GalarSubRegions {
     CrownTundra,
 }
 
+// Gender Types
+export enum Genders {
+    Genderless = 0,
+    MaleFemale, // Default
+}
+
+// Gender Ratio
+export const FEMALE_0 = 0;
+export const FEMALE_12_5 = 0.125;
```

Comment:
The idea behind changing the numbers to something that made sense, was so we no longer needed them as GameConstants :)

Reply:
Removed the constants.